README.md

```diff
@@ -1 +1,61 @@
-# capstone-1
\ No newline at end of file
+# capstone-1
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CAPSTONE-1
+
+>capstone-1
+
+## Built With
+
+- Html&css
+-javascript
+
+
+
+## Live Demo 
+[Live Demo Link](https://monicah79.github.io/capstone-1/)
+
+
+## Project Presentation
+[live Loom Link](https://www.loom.com/share/37541db9052c47bd80e8bb9abe57c678)
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+git & browser
+
+### Setup
+clone repository
+
+## Authors
+
+👤 **monicah wanjema**
+
+- GitHub: [@monicah79](https://github.com/monicah79)
+
+- LinkedIn: [https://www.linkedin.com/in/monica-wanjema-079192193/](https://linkedin.com/in/monica-wanjema-079192193)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used 
+- Inspiration
+- Cindy shin
```

Comment:
- [ ] Kindly notice that the "Acknowledgements" section should give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).